components/script/dom/window.rs

```diff
@@ -120,15 +120,15 @@ pub struct Window {
     #[ignore_heap_size_of = "trait objects are hard"]
     script_chan: MainThreadScriptChan,
     #[ignore_heap_size_of = "thread sources are hard"]
```

Comment:
Not sure if I should also change those annotation descriptions? (thread sources -> task sources) I guess not since the type is a ThreadSource anyway?